test/operator.jl

```diff
@@ -547,9 +547,9 @@ function operators_test(ModelType::Type{<:JuMP.AbstractModel}, VariableRefType::
             @test_throws ErrorException A./y
             @test_throws ErrorException B./y
 
-            @test JuMP.isequal_canonical((2 .* x) ./ 3, Matrix((2 .* y) ./ 3))
-            @test JuMP.isequal_canonical(2 .* (x ./ 3), Matrix(2 * (y ./ 3)))
-            @test JuMP.isequal_canonical((x[1,1],) .* A, Matrix((x[1,1],) .* B))
+            @test JuMP.isequal_canonical((2 .* x) ./ 3, JuMP.densify_with_jump_eltype((2 .* y) ./ 3))
```

Comment:
It's better to structure the tests so that they test only one thing. That may be happening here, but it's hard to tell from reading the test. Is it a test of `JuMP._densify_with_jump_eltype` or of operators on sparse objects?

Reply:
I guess the old test was technically testing both, though I'm not sure of the intention since I didn't write it (just renamed the old pirated constructor). I can factor these out.